### ISSUE TITLE
Eglo Fueva-Z

### ISSUE THREAD

**Issue by cameo77** [dresden-elektronik/deconz-rest-plugin#6044]
## Device

- Product name: Fueva-Z
- Manufacturer: AwoX
- Model identifier: TLSR82xx
- Device type:
  - Light

### Basic

![Eglo_Fueva_Z_2](https://user-images.githubusercontent.com/104138199/167788852-f0a7ed43-6c67-427b-b0b6-a8a8261dc1e8.jpg)
![Eglo_Fueva_Z_12](https://user-images.githubusercontent.com/104138199/167789644-549384d4-749a-479b-8905-2f901ac37c19.jpg)

### Identify

![Eglo_Fueva_Z_3](https://user-images.githubusercontent.com/104138199/167788915-9f507fcd-fc9f-4ca3-8d68-7c66effa9804.jpg)

### Groups

![Eglo_Fueva_Z_4](https://user-images.githubusercontent.com/104138199/167788976-9de49bbf-14b5-4f9f-9890-a65647abee2a.jpg)
![Eglo_Fueva_Z_4_1](https://user-images.githubusercontent.com/104138199/167788992-9628649c-3fa3-4fef-bd74-7f5a0064daf5.jpg)

### Scenes

![Eglo_Fueva_Z_5](https://user-images.githubusercontent.com/104138199/167789095-ebef79ef-e30c-46a1-b7e5-37b230e6b934.jpg)
![Eglo_Fueva_Z_5_1](https://user-images.githubusercontent.com/104138199/167789103-6cb82b9c-fbd4-476b-bb76-28d210093b92.jpg)
![Eglo_Fueva_Z_5_2](https://user-images.githubusercontent.com/104138199/167789109-3ab00561-0374-4b26-871b-f40a48956134.jpg)
![Eglo_Fueva_Z_5_3](https://user-images.githubusercontent.com/104138199/167789143-721e6f70-8a54-4ea0-892e-13ac528df2dd.jpg)

### On/Off

![Eglo_Fueva_Z_6](https://user-images.githubusercontent.com/104138199/167789225-7df4e23e-e357-47de-8a57-b30839b8cd23.jpg)
![Eglo_Fueva_Z_6_1](https://user-images.githubusercontent.com/104138199/167789236-75b277fb-8506-463b-9922-549106f632c5.jpg)
![Eglo_Fueva_Z_6_2](https://user-images.githubusercontent.com/104138199/167789242-90725d56-afd1-46fd-8229-c7267ab74d4d.jpg)

### Level Control

![Eglo_Fueva_Z_7](https://user-images.githubusercontent.com/104138199/167789271-f5ff1f64-ddbf-43d4-ab9d-f1192e18bcb4.jpg)
![Eglo_Fueva_Z_7_1](https://user-images.githubusercontent.com/104138199/167789281-44e72b03-46d3-4ba6-a1d7-144888b03879.jpg)

### Color Control

![Eglo_Fueva_Z_8](https://user-images.githubusercontent.com/104138199/167789305-88a826f9-6642-4176-9ae9-957e00ef61be.jpg)
![Eglo_Fueva_Z_8_1](https://user-images.githubusercontent.com/104138199/167789317-fc0ffc86-bfe5-4790-8eae-1a3aa656954e.jpg)
![Eglo_Fueva_Z_8_2](https://user-images.githubusercontent.com/104138199/167789324-652b7955-5266-40db-8e44-ccfdc203e48f.jpg)
![Eglo_Fueva_Z_8_3](https://user-images.githubusercontent.com/104138199/167789341-1ce01074-5f53-4938-b0f5-75597efe39bc.jpg)
![Eglo_Fueva_Z_8_4](https://user-images.githubusercontent.com/104138199/167789348-31d24c7f-92b0-41b7-b3ea-f4278c138172.jpg)
![Eglo_Fueva_Z_8_5](https://user-images.githubusercontent.com/104138199/167789351-6b043384-4a14-4ede-b1dc-031fc52628f8.jpg)
![Eglo_Fueva_Z_8_6](https://user-images.githubusercontent.com/104138199/167789365-714ab92b-ea6b-40ff-a755-05cdcf64b34a.jpg)
![Eglo_Fueva_Z_8_7](https://user-images.githubusercontent.com/104138199/167789371-9e66b3a6-a759-44ce-98e5-24e4b4c6907f.jpg)
![Eglo_Fueva_Z_8_8](https://user-images.githubusercontent.com/104138199/167789376-de3c48fc-cedc-4e4a-aae9-5e131fff5df5.jpg)
![Eglo_Fueva_Z_8_9](https://user-images.githubusercontent.com/104138199/167789383-0c7715b4-b151-42d7-b608-46e4f805a4f8.jpg)

### ZLL commissioning

![Eglo_Fueva_Z_9](https://user-images.githubusercontent.com/104138199/167789493-7aa7dca1-d759-4ce0-857b-30f04719093e.jpg)
![Eglo_Fueva_Z_9_1](https://user-images.githubusercontent.com/104138199/167789509-e003d5c9-bb6c-4337-8a2f-835ad389580e.jpg)

### Unknown

All Unknown Cluster look the same.

![Eglo_Fueva_Z_10](https://user-images.githubusercontent.com/104138199/167789599-c864f6e2-7182-4be3-867a-4f8572cdc49f.jpg)

**Comment by Mimiix** [comment]
Is there an issue with this light?

**Comment by cameo77** [comment]
Sorry, I should have explain it. My Problem is way simpler than you think. I added this light through the Phoscon Web UI in the Home Assistant Add-On. If I want to control the color temperature, the value range of the slider is larger than the value range of the device. So the slider has a  "dead" zone at the bottom and at the top. As I found out just now, it can be solved by reading in the value range manually in the deconz UI. But is it possible that this works right away through the Phoscon UI? Or is reading in manually the normal way to go? (I'm pretty new to Phoscon/deconz)
Otherwise the device works very well.
Thanks

**Comment by Smanar** [comment]
And you can't use HA to control the temperature ? (will be faster to have a correction on HA than phoscon)

**Comment by cameo77** [comment]
I tried it again more carefully. With the Phoscon UI I can use the full range of the slider. However, no absolute values are displayed there. Only in Home Assistant UI, the slider has two dead zones. I can move the slider to the lowest position, but then it jumps to the value 250, see here:
<img width="275" alt="image" src="https://user-images.githubusercontent.com/104138199/168015173-59bf9553-72d5-4afc-81c1-8ca892cbb4ee.png">

If I slide it to the highest position, it jumps to the value 454. The lights there are acutally warm white and not neutral, as the slider shows:
<img width="281" alt="image" src="https://user-images.githubusercontent.com/104138199/168015071-1582b4ca-8e94-4106-9d2f-203f955f4342.png">

Anyway this can be fixed if I read in the attributes manually via the deconz UI. That would be okay for me too, I know it now.

**Comment by Mimiix** [comment]
@kane610 Would this be something on the device or on the implementation at HA? As the thing manually  goes back.

**Comment by Kane610** [comment]
It depends on if the ct max and min values are reported properly in the rest API. Based on what is described here it sounds like perhaps the range is either wrong or not reported (just a guess though)

The Integration bases its range on what is reported if it exist (else it falls back to default values)

How pydeconz handles ct max/min https://github.com/Kane610/deconz/blob/abfceb1d8511f30854fa099cb38bd689abb3df43/pydeconz/models/light/light.py#L116

How the integration handles ct max/min https://github.com/home-assistant/core/blob/3ce19cd6f870b91f162c69d027a52f2f795f899d/homeassistant/components/deconz/light.py#L291

**Comment by Smanar** [comment]
Yes can you check for the ct min/maxvalue you have in the API ? For exemple using phoscon / help / API Information / light

**Comment by cameo77** [comment]
Okay, I deleted the light from Phoscon and did a reset of the light's hardware. After that I reconnected it through Phoscon. In deconz it looks like this:
![Eglo_Fueva_Z_support_1](https://user-images.githubusercontent.com/104138199/168134907-64b121d1-9b1d-40c7-bd40-e28f59bae9a7.jpg)

In Phoscon (as you described):
`{
    "colorcapabilities": 16,
    "ctmax": 454,
    "ctmin": 250,
    "etag": "c1ec5ef56a9e9237193569d161827b12",
    "hascolor": true,
    "lastannounced": null,
    "lastseen": "2022-05-12T17:27Z",
    "manufacturername": "AwoX",
    "modelid": "TLSR82xx",
    "name": "Color temperature light 5",
    "state": {
        "alert": "none",
        "bri": 254,
        "colormode": "ct",
        "ct": 454,
        "on": true,
        "reachable": true
    },
    "swversion": "0122052017",
    "type": "Color temperature light",
    "uniqueid": "a4:c1:38:bb:f0:4a:08:52-01"
}`
And in Home Assistant I wasn't able to control it. This error message appeared:
![Eglo_Fueva_Z_support_2](https://user-images.githubusercontent.com/104138199/168135323-6910f0d2-aab8-4af4-beac-6e5cf4d69084.jpg)
The manufacturer and the model identifier were unknown in Home Assistant, too.
After a restart of Home Assistant I can control it now, and, however that happened, everything is correct (including the slider in Home Assistant). I don't know what's different this time. I've tried the resetting and reconnecting several times in the last days and never it behaved like that. But I remember, that the restart of Home Assistant wasn't necassary at that tries.
To be complete here is a actual screenshot of the values in deconz:
![Eglo_Fueva_Z_support_3](https://user-images.githubusercontent.com/104138199/168145827-6611416d-419b-4374-9b29-a4abe28a62bc.jpg)

By the way, in the last days I had the same problem I described in the last post with another new light. But I think I will test this again soon.
